src/components/PeopleTable.tsx

```diff
@@ -21,7 +75,14 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Sex
-              <a href="#/people?sort=sex">
+              <a
```

Comment:
You should use the Link component from react-router here as well
Moreover, in this task, you have already created SearchLink that will be useful here